packages/android-foreground-service/src/definitions.ts

```diff
@@ -77,6 +77,23 @@ export interface ForegroundServicePlugin {
    * @since 0.3.0
    */
   requestManageOverlayPermission(): Promise<ManageOverlayPermissionResult>;
+  /**
+   * Create a notification channel.
+   * If not invoked, the plugin creates a channel with name and description set to "Default".
+   *
+   * Only available for Android (SDK 26+).
+   *
+   * @since 6.1.0
+   */
+  createNotificationChannel(options: CreateChannelOptions): Promise<void>;
```

Comment:
```suggestion
  createNotificationChannel(options: CreateNotificationChannelOptions): Promise<void>;
```

Please rename the options type to match the methods name.